api/v3/routes/tracks.rb

```diff
@@ -65,6 +65,14 @@ class Tracks < Core
         implementation.zip.string
       end
 
+      get '/tracks/:id/global' do |id|
+        track = find_track(id)
+        filename = "%s-%s.zip" % [id, "global"]
```

Comment:
Is this line tested? 
What would happen if I changed it to `filename = 'blablabla'?